src/SearchResults.js

```diff
@@ -4,6 +4,7 @@ import { useLocation } from "react-router-dom";
 import { Helmet } from "react-helmet";
 import { SOUND_LIST_QUERY_PARAMS } from "./constants";
 import SoundListContainer from "./SoundListContainer";
+import Paginate from "./components/Paginate.tsx";
```

Comment:
extension shouldn't be necessary

Reply:
I get linter errors without the extension.

![Screen Shot 2020-12-17 at 5 18 47 PM](https://user-images.githubusercontent.com/43425812/102562439-ef0a6600-408b-11eb-9f62-a5a9d22de68f.png)

Reply:
Ah I see. You need to rename `Paginate.tsx` to `Pageinate.js`

Reply:
If I do that, I receive messages saying, "Type annotations can only be used in TypeScript files."

![Screen Shot 2020-12-18 at 10 51 49 AM](https://user-images.githubusercontent.com/43425812/102650008-0a728100-411f-11eb-86ed-ac3d0c275a6b.png)

Reply:
I see. then we have to remove the type annotations. can't use type annotations in a js file.

Reply:
Can we keep the type annotations and leave the filename as it was originally at `Paginate.tsx`? It doesn't cause any issues that I noticed.

Reply:
I see. I think I missed that you were trying to write this in TS